vnext/Scripts/copyRNLibraries.js

```diff
@@ -4,121 +4,115 @@
-  if (fs.existsSync(rnCodeGenDest)) {
-    rimraf.sync(rnCodeGenDest);
-  }
-  const integrationTestsDest = path.resolve(baseDir, 'IntegrationTests');
-  if (fs.existsSync(integrationTestsDest)) {
-    rimraf.sync(integrationTestsDest + path.sep + '*.js');
-  }
-  const librariesDest = path.resolve(baseDir, 'Libraries');
-  if (fs.existsSync(librariesDest)) {
-    rimraf.sync(librariesDest);
-  }
-  const flowDest = path.resolve(baseDir, 'flow');
-  if (fs.existsSync(flowDest)) {
-    rimraf.sync(flowDest);
-  }
-  const flowTypedDest = path.resolve(baseDir, 'flow-typed');
-  if (fs.existsSync(flowTypedDest)) {
-    rimraf.sync(flowTypedDest);
-  }
-  const jestDest = path.resolve(baseDir, 'jest');
-  if (fs.existsSync(jestDest)) {
-    rimraf.sync(jestDest);
-  }
-  const rnTesterDest = path.resolve(baseDir, 'RNTester');
-  if (fs.existsSync(rnTesterDest)) {
-    rimraf.sync(rnTesterDest);
-  }
+    const curSrcPath = path.resolve(srcPath, dirSpec.src);
+    const curTargetPath = path.resolve(targetPath, dirSpec.dest);
 
-  if (fs.existsSync(path.resolve(rnPath, 'IntegrationTests'))) {
-    copyJSFolderRecursiveSync(
-      path.resolve(rnPath, 'IntegrationTests'),
-      baseDir,
-    );
-  }
+    glob.sync('**/*.{js,png,gif,h}', {cwd: curSrcPath}).forEach(file => {
+      const dir = path.dirname(file);
+      const targetDir = path.resolve(curTargetPath, dir);
+      const targetFile = path.resolve(curTargetPath, file);
 
-  if (!fs.existsSync(path.resolve(baseDir, 'packages'))) {
-    fs.mkdirSync(path.resolve(baseDir, 'packages'));
-  }
-  if (!fs.existsSync(path.resolve(baseDir, 'packages/react-native-codegen'))) {
-    fs.mkdirSync(path.resolve(baseDir, 'packages/react-native-codegen'));
-  }
+      fs.mkdirSync(targetDir, {recursive: true});
+      fs.writeFileSync(
+        targetFile,
+        fs.readFileSync(path.join(curSrcPath, file)),
+      );
+    });
+  });
+}
 
-  copyJSFolderRecursiveSync(
-    path.resolve(rnPath, 'packages/react-native-codegen/src'),
-    path.resolve(baseDir, 'packages/react-native-codegen'),
+function copyFile(srcPath, targetPath, filename) {
+  fs.writeFileSync(
+    path.resolve(targetPath, filename),
+    fs.readFileSync(path.resolve(srcPath, filename)),
   );
-  copyJSFolderRecursiveSync(path.resolve(rnPath, 'flow'), baseDir);
-  copyJSFolderRecursiveSync(path.resolve(rnPath, 'flow-typed'), baseDir);
-  copyJSFolderRecursiveSync(path.resolve(rnPath, 'Libraries'), baseDir);
-  copyJSFolderRecursiveSync(path.resolve(rnPath, 'jest'), baseDir);
-  copyJSFolderRecursiveSync(path.resolve(baseDir, 'src/jest'), baseDir); // Copy js files from src/jest to jest
+}
 
-  fs.writeFileSync(
-    path.resolve(baseDir, 'rn-get-polyfills.js'),
-    fs.readFileSync(path.resolve(rnPath, 'rn-get-polyfills.js')),
+exports.copyRNLibraries = () => {
+  const reactNativePath = path.dirname(
+    require.resolve('react-native/package.json'),
   );
+  const reactNativeWinPath = path.resolve(__dirname, '..');
```

Comment:
react-native-win32 uses this script too, and wants it to copy to that folder instead of here.
 -- thats why the baseDir is being passed in.  And probably why your PR is failing in react-native-win32.

Reply:
Oh wow, I completely missed that we were taking in baseDir. That makes a lot of sense. I will fix that.